### DISCUSSION TITLE
subtitles

### DISCUSSION THREAD

**Discussion by satstarter** [jellyfin/jellyfin-androidtv#1159] [General]
there's still no change about subtitles in release 12 thats why i still use kodi
everything else works great i know how to change subtitle size and background in exoplayer
but not in libvlc 
can someone tell me when subtitle setting wil be included
ore wher i find the code for libvlc so i can change the size in libvlc
because in libvlc the srt size is still to large for me

thanx in advance

**Comment by satstarter** [comment]
Is there no one that can tell me where to find the subtitle code for libvlc please??
thanx

**Comment by thornbill** [comment]
I believe subtitles are handled entirely by libVLC when it is used for playback. I have not had much luck finding any documentation for libVLC really. I'm sure there are some options for controlling the appearance, but it probably won't be very easy to find. 😅

**Reply by satstarter** [reply]
Ok thanx for your reply i keep testing and modify the code (for as far as my knowles go :p ) 
at the moment choosing libvlc in jellyfin plays smoother than exoplayer for instance choosing a chapter in exoplayer
is a no go because exoplayer keep on jumping back to the beginning of the video file
but annyway thanx again for your reply ;)
